ui/lib/components/BaseSelect/index.tsx

```diff
@@ -221,7 +221,7 @@ function BaseSelect<T>({
         showAction={[]}
         hideAction={[]}
         popupPlacement="bottomLeft"
-        popupTransitionName="slide-down"
```

Comment:
"slide-down" doesn't work anymore after upgrading antd to 4.16, and it breaks the BaseSelect functions.